onboarding/source/amb_light_service.c

```diff
@@ -1,39 +1,125 @@
 #include "amb_light_service.h"
 #include "serial_io.h"
+#include <stdio.h>
 
 #include <adc.h>
 #include <sci.h>
+#include <FreeRTOS.h>
+#include <os_task.h>
+
+#include <os_queue.h>
+#include <os_projdefs.h>
+#include <string.h>
+
+#define Light_Sensor 	6U
 
 /* USER CODE BEGIN */
 // Include any additional headers and global variables here
 
+static TaskHandle_t lightServiceHandle = NULL;
+static QueueHandle_t queue = NULL;
```

Comment:
I'd make this variable name a bit more descriptive, like `event_queue` or `light_service_queue`.